scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java

```diff
@@ -325,7 +326,13 @@ public void simulateJobStartAndCancelIt(final List<EligibleTaskDescriptor> tasks
         infrastructure.getInternalOperationsThreadPool().submit(new Runnable() {
             public void run() {
                 TerminationData terminationData = jobs.simulateJobStart(tasksToSchedule, errorMsg);
-                terminationData.handleTermination(SchedulingService.this);
+                try {
+                    terminationData.handleTermination(SchedulingService.this);
+                } catch (IOException e) {
```

Comment:
I've done changes, i'll see what sonarqube will say now, thanks @marcocast and @fviale!